### DISCUSSION TITLE
Como vencer a Procastinação?

### DISCUSSION THREAD

**Discussion by grvcoelho** [frontendbr/forum#308] [Discussão]
Oi turma, tudo bem?

Procastinação é um tema que me interessa muito porque me afeta muito e eu acredito que afeta a todos, principalmente quem trabalha com tecnologia. Na semana passada eu dei uma talk aqui no Pagar.me sobre "Por que procastinadores procastinam?" 
- vídeo: [Vídeo da Talk](https://www.youtube.com/watch?v=tMuU6fViaS8)
- slides: [Slides da talk](https://github.com/grvcoelho/lightning/tree/master/procastination)

Minha talk é baseada nestes dois artigos sensacionais do blog Wait But Why:
- [Why Procastinators Procastinate?](http://waitbutwhy.com/2013/10/why-procrastinators-procrastinate.html)
- [How to Beat Procastination](http://waitbutwhy.com/2013/11/how-to-beat-procrastination.html)

E aí queria abrir aqui para um debate:
- Quanto vocês acham que a Procastinação afeta a vida de vocês?
- O que vocês fazem para superar a Procastinação?

**Comment by felipefialho** [comment]
Fala @grvcoelho, no meu caso:
- Tirei todas as notificações do WhatsApp e Messenger. No caso do WhatsApp, desabilitei inclusive nas configurações do celular. Só olho quando necessário.
- Nunca deixo o Messenger ou o WhatsApp Web disponíveis no navegador
- Aos poucos forcei meu cérebro para perder completamente o habito de acessar o Facebook. Só entro lá para divulgar artigos que escrevi
- As únicas redes sociais que deixo acessiveis é o Twitter e o Github, porque na minha concepção, elas ajudam, e nos tempos de "descanso" entre as tarefas, acabo vendo coisas relevantes.
- Nunca deixo trabalho para o outro dia, se tenho uma lista de tarefas a ideia é fazer de tudo para terminar no mesmo dia
- [Pomodoro](https://pt.wikipedia.org/wiki/T%C3%A9cnica_pomodoro) ajuda muito

**Comment by camilatigre** [comment]
Faço algumas muitas coisas que o @LFeh citou: 
- Tiro notificações do WhatsApp e Messenger.
- Não tenho notificações no Desktop
- Uso uma extensão do Chrome, [Kills Feed](https://chrome.google.com/webstore/detail/kill-news-feed/hjobfcedfgohjkaieocljfcppjbkglfd), pra ajudar a fugir do FB. Sem nenhum propósito eu tenho mania de abrir, e se o feed ta disponível vão-se horas lá.
- Twitter e GitHub para descansos úteis, digamos assim.

**Comment by yanmagale** [comment]
Ótimo tema para debate @grvcoelho 
- Quanto vocês acham que a Procastinação afeta a vida de vocês?
  Cara, bastante ... Programação é uma área que requer muito esforço mental. Estar cansado é uma rotina, então acaba que você precisa balancear dias de pouco cansaço e de cansaço extremo. No meu caso, e acredito que de muitos, faculdades ou formações acadêmicas incidem ainda mais. É realmente bem frustrante quando você se planeja para fazer algo e não consegue.
- O que vocês fazem para superar a Procastinação?
  Faço muito das coisas que o @LFeh e a @camilatigre disseram. Acrescento a lista:
- Uma boa playlist (se manter pilhado e motivado a fazer algo ajuda bastante pra mim)
- Para todolists, eu uso o [workflowy.com](Workflowy), de tão simples ele acaba sendo um dos melhores na minha opinião
- Tenho instalado um app chamado "Goal Tracker", nele consigo acompanhar diariamente meu progresso em cada meta/tarefa. Isso me permite "evitar" os ciclos de procrastinação, me forçando a fazer as tarefas que eu realmente queria e precisava

**Comment by fabiootc** [comment]
Muito bom! 

Acredito que a Procastinação não afeta diretamente o meu dia a dia no trabalho (até porque meu atual ritimo é muito frenético pra isso), mas me afeta muito no que diz respeito a evoluir didaticamente em 
em algo que preciso aprender e não me divirto. Por exemplo, acho bem "boring" praticar meu Inglês diariamente e nesses momentos, quase sempre perco a paciência e mudo meu foco acabando no final estudando outras paradas.

Não tenho uma prática regular para superar a proscatinação, o que me tira disso nestas situações é o próprio "bitch slap" da vida mesmo. ;)

**Comment by theandersonn** [comment]
Gostei da talk @grvcoelho grv, parabéns!

Já tive alguns momentos de procrastinação tão severos que me prejudiquei profissionalmente.

Tomei algumas medidas, talvez drasticas... como cancelar conta no Facebook [**não me arrependo**], fico offline no smartphone e em tempos regulares acesso e interajo, respondo quando necessário.

Hoje Redes Sociais se resumem em: Slack, GitHub e Twitter.

Já fiz um período de pomodoro, quando consegui dominio de tempo nas execuções de tarefas deixei um pouco de lado, mas mentalmente sigo a metodologia.

Não é fácil, me cobro muito. 
- Playlist maneira pra dar um gás e não ouvir conversa em torno da minha mesa;
- Abro vários cards no trello e modularizo as demandas em pequenas tarefas;
- Estabeleço metas razoáveis que possam ser alcançadas diariamente e semanalmente.
- Aproveito todo tempo possível, revertendo em tempo util.

**Comment by felipefialho** [comment]
@theandersonn Também deixei o Pomodoro meio de lado depois de adquirir certo padrão na resolução das tarefas.

**Comment by DevCarlosAugusto** [comment]
Caras, eu tenho testado atualmente o Wakatime[https://wakatime.com/] no meus editores[atualmente uso o Atom e o WebStorm] para saber o Ritmo e tempo que tenho levado em cada tarefa na programação. Desinstalei o facebook do celular e tenho tirado vários outros apps também, mas sofro do mesmo problema que o @fabiootc disse, termino achando algumas coisas chatas e partindo para outras tarefas muitas vezes....

**Comment by paoloalmeida** [comment]
Creio que isso é um problema bem comum. Ainda mais pela avalanche de informações que temos hoje em dia, seja no computador ou celular... A parada é saber digerir e administrar isso, junto com as tarefas.
Uso alguns Apps/Métodos que ajudam:
[RescueTime](https://www.rescuetime.com/) - Que me dá um overview geral, onde e como gasto meu tempo dentro da minha máquina. Ele categoriza automagicamente, se o App aberto é algo do seu workflow, ou não.
[Wakatime](http://wakatime.com/) - Me dá uma noção mais assertiva, sobre projetos que perdi mais tempo, linguagem/editores que uso com mais frequência.
[FocusBooster](https://www.focusboosterapp.com/) - To testando ainda, mas é basicamente Pomodoro Technique, com um dashboard de métricas. Me parece bem bacana.
Do not Disturb - Função mágica do mac, que não deixa aparecer nenhuma notificação no desktop. Ajuda absurdamente.
[Bullet Journal](http://bulletjournal.com/get-started/) - Mais do que qualquer outro método, isso funciona ridiculamente pra mim. Famoso papel e caneta :)

Muito bom tópico @grvcoelho :)

**Comment by talitapagani** [comment]
Ótimo tópico, @grvcoelho! Esse é um assunto muito importante pois nem sempre nos damos conta que procrastinamos e como isso é sabotador para nossa produtividade.

Gostei das dicas do @LFeh sobre notificações, esse ainda é meu pecado!

Mas algumas coisas que eu faço pra vencer a vontade de deitar e dormir eternamente quando tenho várias coisas pra fazer são:
- Faço sempre uma lista de tarefas diárias e outra semanal do que tenho que realizar e sempre tentando quebrar assuntos grandes em pequenas tarefas. Isso ajuda muito a entender por onde eu tenho que começar e permite ter um senso de realização maior. Quando pegamos uma tarefa imensa e não conseguimos cumprir, podemos nos sentir improdutivos ou não ter vontade de fazer porque aquilo é complexo, mas se quebrarmos ela, conseguimos ver que já realizamos algo e estamos no caminho;
- Uso extensões como o [Focus45 ](https://chrome.google.com/webstore/detail/focus-45/efjdfkfpnffgkdgehkenoikpbadfgple) para bloquear o acesso a determinados sites que eu configuro, pois admito que não tenho autocontrole às vezes;
- Já usei também o [PomodoroApp ](http://www.teamviz.com/pomodoro-app/) para gerenciar a lista de tarefa e a execução do pomodoro para cada tarefa;
- Tenho o problema de pensar demais na tarefa e nas suas implicações antes de começar, o famoso _analysis paralysis_. Aí, para algumas tarefas específicas, comecei a suprimir o planejamento e sento logo pra fazer, aí eu vou pensando progressivamente conforme as coisas vão acontecendo (ex.: pendências de escrita que tenho que fazer para minha dissertação de mestrado -> pego pra fazer mesmo sem ter esquematizado o como fazer). Como dizem, uma coisa de cada vez;
- Já usei também no trabalho a técnica de matrix de prioridades (_priority matrix_), que apresenta os quadrantes importante e urgente, importante e não urgente, não importante e urgente, não importante e não urgente. Me ajudou horrores a estabelecer melhor o que tinha que fazer e até quais e-mails responder.

Espero ter ajudado :)

**Comment by lagden** [comment]
Programo por hobby, so I can go with the flow!!
Tanto que acho que a técnica de pomodoro totalmente bullshit...
Pense comigo... vc está no flow e derrepente times up!! bahhh!!! igual correr a são silvestre e parar pra descansar 3 minutos...

Faça por prazer...
parece até que sou um irresponsável, sqn!!!
apenas não tenho religião, logo none procrastinação!!

@lagden para Presidente!!! 😂

**Comment by felipefialho** [comment]
@lagden HAHAHAA, cara...

A técnica de Pomodoro funciona muito bem para você se habituar a ter foco. Mas depois que vira algo natural, também não vejo tanta utilidade assim.

**Comment by lagden** [comment]
Muito DDA e pouco TDD!!!

**Comment by talitapagani** [comment]
Acho interessante a técnica Pomodoro para algumas tarefas. A procrastinação atinge acho que mais as tarefas que não são relacionadas à programação, pois geralmente você já possui um escopo mais claro do que fazer. Ela é útil para ajudar na concentração ao resolver tarefas que precisam de foco e de um impulso para você começar e entrar no ritmo. Tem uns estudos interessantes que mostram que as pessoas até continuam a trabalhar no que estavam fazendo depois de terminado o Pomodoro, pois elas conseguiram engrenar na tarefa. Mas sempre a técnica mais efetiva é a que funciona para a pessoa, não existem regras.

**Comment by suissa** [comment]
A resposta é:

![](https://img.ifcdn.com/images/1d62439bf2dc237bc673374c8acc4a0ddc65b290ab4b6bbfe83b3c535023ca84_1.jpg)

**Comment by santosluiz** [comment]
Utilizo técnicas semelhantes ao do @LFeh e da @camilatigre.
- Desinstalei o Facebook e o Messenger do meu celular. Só acesso o facebook pelo navegador do cel mesmo. Messenger, somente se for algo importante eu vou para o pc e respondo;
- Quando eu vou codar, me desconecto de tudo. Procuro um ambiente mais silencioso possível e deixo o celular no modo avião! Nada de whatsapp, facebook e telegram rsrs.
- Faço uma espécie de ToDo com todas as minhas tarefas do dia e as divido em tarefas menores. "Se eu preciso criar 5 telas, cada tela é uma tarefa". Não tem sensação melhor do que terminar algo e arrancar o post-it dali!
- Utilizo também a técnica do Pomodoro. Não necessariamente cronometrando o tempo, mas sim estabelecendo objetivos. "Após eu terminar esta coisa, irei beber uma água"...

Utilizando essas técnicas e estipulando o meu [senso de urgência](http://mundoraiam.com/vagabundo/)(esse texto tem uma abordagem bem diferente e me ajudou muito) consegui evoluir bastante!
